starter-core/src/test/groovy/io/micronaut/starter/feature/messaging/rabbitmq/RabbitMQSpec.groovy

```diff
@@ -46,7 +46,7 @@ class RabbitMQSpec extends ApplicationContextSpec implements CommandOutputFixtur
         template.contains('''<artifactId>micronaut-maven-plugin</artifactId>
           <configuration>
             <shared>true</shared>
-          </configuration>''')
```

Comment:
I had to remove this because config with AOT enabled looks like: 

```xml
    <artifactId>micronaut-maven-plugin</artifactId>
          <configuration>
            <shared>true</shared>
            <configFile>aot-${packaging}.properties</configFile>
          </configuration>
```